### DISCUSSION TITLE
Timed objects on watchface

### DISCUSSION THREAD

**Discussion by ghost** [zepp-health/zeppos-forum#201] [Q&A] [answered]
Hi All,

I've recently (a few months ago now) purchased an Amazfit GTS 4 mini, mainly to be able to create my own watchfaces and customize the device to my liking. In the process I've created a growing list of watchfaces, trying different approaches. 
I would now like to be able to display information on the watch face on certain days, ie some days of the weeks I'd like text/images visible on the watchface, but not on other days, and the text will vary for days. Other information I'd like to display based on time of day (and ideally also depending on day of week) starting at a time point and disappearing again after a set time. Is this possible, either by injecting code into the watchface, or by using a mini app?
Is it possible to have an object rotate based on 24 hours? Instead of a 12 hour analog pointer, a 24 hour analog pointer?

Thank you!

**Comment by SashaCX75** [comment, marked as answer]
Yes it is all possible. Here's an example with the 24-hour analogue time format.
[BOTTA_TRES_24(AOD).zip](https://github.com/zepp-health/zeppos-forum/files/11859167/BOTTA_TRES_24.AOD.zip)

Everything else you want to do can be done with the time sensor.
https://docs.zepp.com/docs/1.0/watchface/api/hmSensor/sensorId/TIME/
Knowing the time and day of the week you can switch the visibility of individual elements on and off as you wish.

**Reply by ghost** [reply]
Thank you!
